File: Backend/src/Repos/Models/XML_Model/DsBenhNhanLoiMaMay.js
const mongoose = require('mongoose');

const DsBenhNhanLoiMaMaySchema = new mongoose.Schema({
  Ma_Lk:     { type: String, default: null },
  Ma_May:    { type: String, default: null },
  ThoiGian:  { type: String, default: null }
}, { 
  timestamps: true,
  strict: false
});

module.exports = mongoose.model('DsBenhNhanLoiMaMay', DsBenhNhanLoiMaMaySchema);

File: WPF_GiamDinhBaoHiemYTe/Repos/Model/Patietn_XML_data/DsBenhNhanLoiMaMay.cs
using CommunityToolkit.Mvvm.ComponentModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPF_GiamDinhBaoHiem.Repos.Model.Patietn_XML_data
{
    
    public partial class DsBenhNhanLoiMaMay : ObservableObject
    {
        
         [ObservableProperty] private string? ma_Lk;
        [ObservableProperty] private string? ma_May;
        [ObservableProperty] private string? thoiGian;
       
        

    }
}
